Jenkinsfile

```diff
@@ -37,23 +37,25 @@ spec:
       deploymentSize: 1
       web:
         enabled: false
-  elasticsearch_manifest: |
-    apiVersion: elasticsearch.k8s.elastic.co/v1beta1
+  elasticsearchManifest: |
+    apiVersion: elasticsearch.k8s.elastic.co/v1
     kind: Elasticsearch
     metadata:
       name: elasticsearch
       namespace: $namespace
     spec:
       version: 7.10.2
+      volumeClaimDeletePolicy: DeleteOnScaledownAndClusterDeletion
```

Comment:
In case you're wondering why this is different, it's because we should remove the PVs when the cluster is deleted in CI otherwise we'll just run out of volumes at some point. Not an issue _right now_ because we use ephemeral storage in CI, but in the future if we move this to a "real" cluster, there is no reason we can't use PVs.